cerebrium/scaling/scaling-apps.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Scaling Apps"
+description: "Learn to optimise for cost and performance by scaling out apps"
+---
+
+Cerebrium's scaling system automatically manages computing resources to match app demand. The system handles everything from a few simple requests, to the processing of multiple requests simultaneously, while optimizing for both performance and cost.
+
+## How Autoscaling Works
+
+The scaling system monitors two key metrics to make scaling decisions:
```

Comment:
maybe mention something in this paragraph that its configurable by them based on traffic